src/main/java/cz/startnet/utils/pgdiff/PgDiffUtils.java

```diff
@@ -527,6 +527,26 @@ public static String getQuotedName(final String name,
     public static String getQuotedName(final String name) {
         return getQuotedName(name, false);
     }
+    
+    public static String getDropIfExists (final boolean dropIfExists){
```

Comment:
This is the heart of your PR right here.  Let's find a way to get rid of all the noise.